### ISSUE TITLE
gitProtocol setting (ssh) is ignored (regr?)

### ISSUE THREAD

**Issue by aetonsi** [microsoft/vscode-remote-repositories-github#334]
Hi again,
like in #294 , i have `"remoteHub.gitProtocol": "ssh"`+`"github.gitProtocol": "ssh"`, but the setting is ignored when i run the command "remote repositories: continue working in new local clone".
As of today (`Remote Repositories v0.35.2023052901 Pre-Release` + `vscode Version: 1.78.2`/`vscode Version: 1.79.0-insider`), using that command opens a new window, the window opens a URL like
> vscode://vscode.git/clone?url%3Dhttps%253A%252F%252Fgithub.com%252Fshaka-project%252Fshaka-player%26ref%3Dmain

then it errors out because on my system HTTPS has been disabled while SSH is enabled:
```log
2023-05-30 11:09:03.738 [info] Log level: Info
2023-05-30 11:09:03.738 [info] Validating found git in: "C:\Program Files\Git\cmd\git.exe"
2023-05-30 11:09:03.867 [info] Using git "2.40.1.windows.1" from "C:\Program Files\Git\cmd\git.exe"
2023-05-30 11:09:04.161 [info] GitProtocolHandler.handleUri(vscode://vscode.git/clone?url%3Dhttps%253A%252F%252Fgithub.com%252Fshaka-project%252Fshaka-player%26ref%3Dmain)
2023-05-30 11:09:05.790 [info] Executing git.clone for https://github.com/shaka-project/shaka-player
2023-05-30 11:09:11.464 [info] > git clone https://github.com/shaka-project/shaka-player c:\_projects\~rnd\shaka-player --progress --branch main [124ms]
2023-05-30 11:09:11.464 [info] Cloning into 'c:\_projects\~rnd\shaka-player'...
fatal: transport 'https' not allowed
```

i think it could be a regression.

---

have a nice day

**Comment by meganrogge** [comment]
@joyceerhl how do I trigger `remote repositories: continue working in new local clone`?

**Comment by joyceerhl** [comment]
Steps:
1. Configure `"remoteHub.gitProtocol": "ssh"` in a remote repository (https://insiders.vscode.dev/github/microsoft/vscode for e.g.)
2. From remote indicator, select Continue Working in New Local Clone
3. This will launch the local VS Code window and prompt for a folder to clone into
4. Open the Git extension log and verify that the clone URL is an SSH URL rather than a HTTP URL

**Comment by bhavyaus** [comment]
<img width="579" alt="image" src="https://github.com/microsoft/vscode-remote-repositories-github/assets/25044782/24b4b5d7-5d40-4194-8b14-bba1616c03b6">

Verifying ssh protocol was used after changing the setting.

**Comment by aetonsi** [comment]
can confirm fixed in `Github Repositories v0.57.2023060101`
